exporter/common.go

```diff
@@ -201,6 +203,22 @@ func listAllCollections(ctx context.Context, client *mongo.Client, filterInNames
 	return namespaces, nil
 }
 
+func filterSystemCollections(cols []string) []string {
+	systemCollections := map[string]bool{
```

Comment:
could you please move out initialization of this map? I'm not sure that it makes sense to create every time this map when `filterSystemCollections` is called.